dbt_sugar/core/task/base.py

```diff
@@ -213,7 +215,9 @@ def update_model_description_test_tags(
         )
 
     def update_column_description_from_schema(
-        self, path_file: Path, dict_column_description_to_update: Dict[str, Dict[str, Any]]
+        self,
```

Comment:
this project allows for a line length of 100 chars. I would rather leave the pre-commit hook take care of the formatting as it will standardise it for everyone. Could you please revert this change?